apps/mobile/src/features/browser/browser/search-bar/search-input.tsx

```diff
@@ -25,34 +32,51 @@ export function SearchInput({
 }: SearchInputProps) {
   const theme = useTheme<Theme>();
   return (
-    <TextInput
-      ref={textInputRef}
-      onFocus={() => {
-        isUrlFocused.value = true;
-      }}
-      onBlur={() => {
-        KeyboardController.dismiss();
-        isUrlFocused.value = false;
-      }}
-      keyboardType="web-search"
-      textContentType="URL"
-      autoCapitalize="none"
-      onChangeText={setTextUrl}
-      value={textUrl}
-      autoComplete="url"
-      onSubmitEditing={onSubmit}
-      width="100%"
-      py="5"
-      px="4"
-      borderRadius="sm"
-      borderColor="ink.border-default"
-      borderWidth={1}
-      placeholderTextColor={theme.colors['ink.text-subdued']}
-      color="ink.text-primary"
-      placeholder={t({
-        id: 'browser.input_placeholder',
-        message: 'Type URL or search',
-      })}
-    />
+    <Box>
+      <TextInput
+        ref={textInputRef}
+        onFocus={() => {
+          isUrlFocused.value = true;
+        }}
+        onBlur={() => {
+          KeyboardController.dismiss();
+          isUrlFocused.value = false;
+        }}
+        keyboardType="web-search"
+        textContentType="URL"
+        autoCapitalize="none"
+        onChangeText={setTextUrl}
+        value={textUrl}
+        autoComplete="url"
+        onSubmitEditing={onSubmit}
+        width="100%"
+        py="5"
+        px="4"
+        borderRadius="sm"
+        borderColor="ink.border-default"
+        borderWidth={1}
+        placeholderTextColor={theme.colors['ink.text-subdued']}
+        color="ink.text-primary"
+        placeholder={t({
+          id: 'browser.input_placeholder',
+          message: 'Type URL or search',
+        })}
+      />
+      <Pressable
```

Comment:
Added an IconButton component few days ago that you can use here, see similar example usage in [here](https://github.com/leather-io/mono/blob/fb4c703af9c8394e3fae45a5e04ef5716d240b1f/apps/mobile/src/features/send/components/recipient/recipient-input.tsx)